workbench-core/datasets/src/dataSetService.ts

```diff
@@ -146,7 +165,24 @@ export class DataSetService {
    * @returns - the DataSet object associated with that DataSet.
    */
   public async getDataSet(dataSetId: string): Promise<DataSet> {
-    return await this._dbProvider.getDataSetMetadata(dataSetId);
+    const metadata: Metadata = {
+      actor: { id: '-', roles: [] },
+      action: this.getDataSet.name,
+      source: {
+        serviceName: DataSetService.name
+      },
+      requestBody: dataSetId
```

Comment:
Is `dataSetId` a requestBody value even if the param is coming from the url and not the body?

Reply:
changing to params